analytic_engine/src/meta/details.rs

```diff
@@ -1317,73 +1250,59 @@ mod tests {
 
     #[derive(Debug)]
     struct MemLogStore {
```

Comment:
Extract `MemLogStore` into a separate file.

Reply:
This belongs to the test module, so no need to set it in a separate file.